frontend/src/components/VisitRouter/VisitForm.js

```diff
@@ -81,13 +81,17 @@ const VisitForm = ({
         </FormControl>
       </Grid>
       <Grid item xs={12}>
-        <TextField
-          fullWidth
-          label="Add a Note"
-          name="notes"
-          value={visitInfo.notes}
-          onChange={e => handleVisitChange(e.target)}
-        />
+        <FormControl>
+          <InputLabel htmlFor="addANote">Add a Note</InputLabel>
+          <PrevPointInput
+            fullWidth
+            multiline="true"
```

Comment:
I think you can just put 'multiline' by itself(not equals true). like the fullwidth flag above.

Reply:
Sure enough.